### PR TITLE
[FIX] typos and minor changes

### PR DESCRIPTION
**Pull request by MaxHillebrand**
standardize on capitalization, and a couple other minor changes.

[WIP] I won't merge it just yet, will probably find some more minor nitpicks that can be done in one PR.

### CODE REVIEW

File: docs/glossary/Glossary.md [hunk 1]
```diff
@@ -296,5 +295,5 @@ Read more: [Technical Overview of Wasabi Wallet](/building-wasabi/TechnicalOverv
 :::details
 ### Wasabika
```
Comment (SimoneBovi) [inline docs/glossary/Glossary.md]: ### Wasabikas

Comment (MaxHillebrand) [reply docs/glossary/Glossary.md]: I think the singular is alright.

File: docs/glossary/Glossary.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ Read more: [Transaction surveillance companies attempting to destroy fungibility
 :::
 
 :::details
-### Hardware Wallet (HW)
+### Hardware Wallet (HWW)
 A hardware wallet is a special type of Bitcoin wallet which generates and stores the user's private keys in a tailor made hardware device.
```
Comment (raindogdance) [inline docs/glossary/Glossary.md]: > stores the user's private keys in a tailor made hardware device.  

This sounds like the device is being manufactured on specific request by the user. I would change it to: 

> stores the user's private keys on a dedicated hardware device.

File: docs/glossary/Glossary.md [hunk 3]
```diff
@@ -205,8 +205,7 @@ Read more: [Wasabi Wallet under the hood](/FAQ/FAQ-UseWasabi.md#how-does-my-wall
 ### Dust
 
 Dust is an UTXO that is uneconomical to spend.
-Also, small portions of bitcoin can lead to serious consequences about one's privacy.
-An example would be the so called `forced address reuse attack`.
+Also, small portions of bitcoin can lead to serious consequences about one's privacy, for example the so called `forced address reuse attack`.
```
Comment (raindogdance) [inline docs/glossary/Glossary.md]: > serious consequences about one's privacy  

should be changed to: 

> serious consequences for one's privacy

File: docs/glossary/Glossary.md (hunk 2 above)
Comment (MaxHillebrand) [reply docs/glossary/Glossary.md]:
```suggestion
A hardware wallet is a special type of Bitcoin wallet which generates and stores the user's private keys on a dedicated hardware device.
```

File: docs/glossary/Glossary.md (hunk 3 above)
Comment (MaxHillebrand) [reply docs/glossary/Glossary.md]:
```suggestion
Also, small portions of bitcoin can lead to serious consequences for one's privacy, for example the so called `forced address reuse attack`.
```

**Pull request merged**